web-client/src/theme/variables.scss

```
@@ -8,69 +8,68 @@
 
 /** Ionic CSS Variables **/
 :root {
```

We should probably link the UI Kit here, for easy reference while maintaining:

```suggestion
:root {
  // See Nautilus UI Kit:
  // https://www.figma.com/file/zYVHu1EwCIDCbQp8rFscyA/Nautilus-Branding?node-id=23%3A128

```